lib/bundler/cli.rb

```diff
@@ -164,28 +168,28 @@ def check
       "Specify the number of jobs to run in parallel"
     method_option "local", :type => :boolean, :banner =>
       "Do not attempt to fetch gems remotely and use the gem cache instead"
-    method_option "no-cache", :type => :boolean, :banner =>
+    deprecated_option "no-cache", :type => :boolean, :banner =>
       "Don't update the existing gem cache."
     method_option "force", :type => :boolean, :banner =>
       "Force downloading every gem."
```

Comment:
I feel like `force` is a very unclear name for this option, and I'd like to deprecate it in favor of something like `--redownload`. Or maybe better yet, a completely different command that nukes the cache, forcing downloads next time. (The latter is out of scope for this PR, though).